android-release.sh

```
@@ -0,0 +1,40 @@
+        export MARCH_FLAG="-march=armv8-a+crc -mtune=cortex-a78"
+    else
+        export MARCH_FLAG="-march=x86-64-v2"
+    fi
+fi
+
+export CONTAINER_NAME="bun-webkit-linux-android-${ANDROID_ARCH}"
+if [ "$WEBKIT_RELEASE_TYPE" == "Debug" ]; then
+    CONTAINER_NAME="${CONTAINER_NAME}-debug"
+fi
+
+mkdir -p "$temp"
```

Shellcheck (0.11.0)

[warning] temp is referenced but not assigned.

(SC2154)